plugins/adr/README.md

```diff
@@ -69,18 +69,69 @@ Afterwards, add the following code snippet to use `AdrSearchResultListItem` when
 ```tsx
 // In packages/app/src/components/search/SearchPage.tsx
 import { AdrSearchResultListItem } from '@backstage/plugin-adr';
+import { AdrDocument } from '@backstage/plugin-adr-common';
 
 ...
 
-case 'adr':
-  return (
-    <AdrSearchResultListItem
-      key={document.location}
-      result={document}
+<SearchType.Accordion
+  name="Result Type"
+  defaultValue="software-catalog"
+  types={[
+    ...
+    {
+      value: 'adr',
+      name: 'Architecture Decision Records',
+      icon: <DocsIcon />,
+    },
+  ]}
+/>
+  
+...
+
+// In filters
+<Paper className={classes.filters}>
+
+...
+  // ADR specific type
+  {types.includes('adr') && (
+    <SearchFilter.Select
+      className={classes.filter}
+      label="Entity"
+      name="name"
+      values={async () => {
+        // Return a list of entities which have ADRs.
+        const { items } = await catalogApi.getEntities({
+          fields: ['metadata.name'],
```

Comment:
I don't think this is correct, I select the entity in the drop-down list and get no matches.

Reply:
ah sorry I missed this, it looks like the `name` prop is defined incorrectly. This should be the name of the specific field in the search result you want to filter on. In this case, since you want to filter on the selected entity, you probably want to filter on the [`entityRef`](https://github.com/backstage/backstage/blob/master/plugins/adr-backend/src/search/createMadrParser.ts#L73) field and in this function you would have to return the full entity ref for each entity instead of just the name for it to filter correctly. Unfortunately that would result in the select value not being very display friendly since it doesn't look like the `SearchFilter.Select` component supports supplying an alternative display label for values 🙁

Reply:
Thinking about this again, maybe we shouldn't include the `SearchType.Accordion` and `SearchFilter.select` examples in the instructions here since it's up to the consume how they want to personalize their search page and this is not specific to this plugin.

Reply:
I left the Accordion with an Optional comment as it does work, but removed the select as that wasn't working.